compiler/damlc/tests/BUILD.bazel

```diff
@@ -453,8 +453,11 @@ da_haskell_test(
         "//compiler/damlc",
         "@damlc_legacy",
         "//compiler/damlc/tests:generate-simple-dalf",
+        "//daml-script/daml:daml-script.dar",
         # Feel free to update this to 0.13.55 once that is frozen.
         ":dars/old-proj-0.13.55-snapshot.20200309.3401.0.6f8c3ad8-1.8.dar",
+        # Used in regression test for https://github.com/digital-asset/daml/issues/14291
+        ":dars/lib-with-script-0.0.1-sdk-2.2.0-lf-1.14.dar",
```

Comment:
How complicated would be to generate a dar to reproduced the regression by using the lf encoder ?

Reply:
the problem is that we need a dar with a copy of `daml-script` from an older SDK, I think at the very least we would need a new bazel rule to fetch `daml-script` from a given SDK

Reply:
:+1: